src/emulate_unified_sdk.c

```diff
@@ -73,7 +73,7 @@ int emulate_syscall_nbgl(unsigned long syscall, unsigned long *parameters,
 {
   (void)version;
 
-  if (model != MODEL_STAX) {
+  if ((model != MODEL_STAX) && (model != MODEL_NANO_SP) && (model != MODEL_NANO_X)) {
```

Comment:
I'm wondering if we shouldn't add some check on API_LEVEL version? Like:
```suggestion
  if (model != MODEL_STAX) && (version < SDK_API_LEVEL_15) {
    return SYSCALL_NOT_HANDLED;
  }
```

Reply:
Agree :)